selfdrive/car/CARS_template.md

```diff
@@ -1,30 +1,17 @@
 {% set footnote_tag = '[<sup>{}</sup>](#footnotes)' -%}
-{% set star_icon = '[![star](assets/icon-star-{}.svg)](##)' -%}
 
 <!--- AUTOGENERATED FROM selfdrive/car/CARS_template.md, DO NOT EDIT. --->
 
 # Supported Cars
 
 A supported vehicle is one that just works when you install a comma device. Every car performs differently with openpilot, but all supported cars should provide a better experience than any stock system.
 
-## How We Rate The Cars
```

Comment:
Maybe a short hand-written description for each would be good. For example what steering torque units are, and what it means/is good. What you get with openpilot ACC vs. Stock ACC